### PR TITLE
Corrected markup in ONBOARDING.md, added codeprm to TEAM

### PR DESCRIPTION
**Pull request by codeprm**

### CODE REVIEW

**Comment by Sheetal777** [comment]
@johnjacobkenny This one looks good to be merged!

**Review by thunder-coding — APPROVED** [review]
Took some time to review.

Please anyone else who is reviewing this take your time. There are a lot of good changes.

File: firstpr/pr_m.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Prathap
+
+CS Student. Now a working professional for a big bank.
+
+I never got a chance to work on open source or UI dev. So, trying to improve my skills through this project.
```
Comment (UnevenCoder) [inline firstpr/pr_m.md:5]: before the first full stop insert 'project'

File: docs/ONBOARDING.md [hunk 2]
```diff
@@ -45,7 +46,7 @@ Once you are part of the twindle team, please join us here to keep in touch.
 4. Do the third PR
 
 **What is the third PR about?**
-- Make few changes in the file `TEAM.md` inside the `docs` directory, commit, through the CLI (command line interface) and raise your second Pull Request.  
+- Make few changes in the file `TEAM.md` inside the `docs` directory, commit, through the CLI (command line interface) and raise your third Pull Request.
```
Comment (UnevenCoder) [inline docs/ONBOARDING.md:49]: This was good error catch good job 👍

**Review by UnevenCoder — CHANGES_REQUESTED** [review]
One change needed and all good 😃

File: firstpr/pr_m.md (hunk 1 above)
Comment (codeprm) [reply firstpr/pr_m.md]: Thanks, file is updated

**Review by Akshay2996 — APPROVED** [review]
All the changes done looks good to me.

**Review by UnevenCoder — APPROVED** [review]
Good to go

**Pull request merged**